cli/tests/testdata/compat/test_runner/top_level_fail_cjs.out

```diff
@@ -0,0 +1,13 @@
+
+test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out ([WILDCARD])
+
+error: Uncaught (in promise) AssertionError: Values are not strictly equal:
+
+
+    [Diff] Actual / Expected
+
+
+-   10
++   20
+
+[WILDCARD]
```

Comment:
Something for another PR, but I found this output very confusing... We should be able to catch these errors before printing test report

Reply:
Confusing how?

Reply:
Confusing in that the error is returned after the test report - so at first glance it seems something failed after running all tests, which most often is not true - usually it's an error in top level code (or floating promise) that happens before tests are run. Again, something to address in a separate PR